src/components/Tweet/Tweet.jsx

```diff
@@ -0,0 +1,116 @@
+import { useState } from 'react'
+import './Tweet.scss'
+import './Tweet-extention.scss'
```

Comment:
try to use scss modules, import like import s from "./Tweet.module.scss"
And use like div className = {s.className}